src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/AssetsFileAttachedCollectionSourceBase.cs

```diff
@@ -142,6 +144,12 @@ protected static void ProcessLibraryContent(ref List<object>? items, AssetsFileT
                 items ??= new List<object>();
                 items.Add(PackageAssemblyGroupItem.CreateWithContainsItems(snapshot, library, PackageAssemblyGroupType.Framework, library.FrameworkAssemblies));
             }
+
+            if (!library.ContentFiles.IsEmpty)
+            {
+                items ??= new List<object>();
+                items.Add(PackageContentFilesGroupItem.CreateWithContainsItems(_fileIconProvider, library.ContentFiles));
```

Comment:
Do a language check similar to https://github.com/dotnet/NuGet.BuildTasks/blob/5244c490a425353ac12445567d87d674ae118836/src/Microsoft.NuGet.Build.Tasks/ResolveNuGetPackageAssets.cs#L572-L575

Reply:
Thanks for confirming the logic. I've added this to the backlog as it's not immediately clear how I plumb the project language through to where it's needed without adding a lot of mess. I'm working on a slightly different design that should make this kind of thing easier (the file icon provider is another example from this PR).